services/course-material/src/components/ContentRenderer/moocfi/ExerciseBlock/ExerciseTask.tsx

```diff
@@ -45,28 +45,32 @@ const ExerciseTask: React.FC<React.PropsWithChildren<ExerciseTaskProps>> = ({
   const cannotAnswerButNoSubmission =
     !canPostSubmission && !exerciseTask.previous_submission && signedIn
 
+    const hasInstruction = currentExerciseTaskAssignment.length > 0 &&
```

Comment:
check if all blocks are paragraphs and if all of them are empty

Reply:
Will this not be counterproductive because the goal of the check is to ascertain if the teacher left the block empty - why do we need to check all paragraphs, or am I missing something?

Reply:
The reason was that the teacher could input a couple of empty lines and forget them there. In that case the instructions would look empty and we should hide them.

Reply:
Okay noted, we were thinking of two different thing - but now I get your point